src/Controllers/ESRTInventoryController.php

```diff
@@ -73,16 +73,20 @@ public function index($request, $response)
                     // }
                 } else {
                     $lightColour = '#FF0000';
-                    $this->logger->info("esrt_inventory '/ifttt/v1/triggers/esrt_inventory' Inserted new event - success");
-                    $this->db->table('esrt_inventory_record')->insertGetId(array(
-                        'title' => $currentCentre["name"],
-                        'description' => "More cots needed at ",
-                        'date_created' => date('Y-m-d H:i:s')
-                    ));
-                    $this->db->table('esrt_inventory_record')
-                    ->orderBy('date_created', 'asc')
-                    ->limit(1)
-                    ->delete();
+                    if ($esrtr[0]->title != $currentCentre["name"]) {
+                        $this->logger->info("esrt_inventory '/ifttt/v1/triggers/esrt_inventory' Inserted new event - success");
+                        $this->db->table('esrt_inventory_record')->insertGetId(array(
+                            'title' => $currentCentre["name"],
+                            'description' => "More cots needed at ",
+                            'date_created' => date('Y-m-d H:i:s')
+                        ));
+                        $this->db->table('esrt_inventory_record')
```

Comment:
I think this should be selecting by the title as well. We will need to support multiple centers in the future.